projects/plugins/wpcomsh/composer.json

```diff
@@ -120,20 +120,25 @@
 	"minimum-stability": "dev",
 	"prefer-stable": true,
 	"config": {
+		"platform": {
+			"php": "7.4"
+		},
```

Comment:
Don't set this, it forces 7.4-compatible versions of phpunit and such to be installed when running 8.x tests.

Reply:
Removed, updated the lock, let's see what happens :)